src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,114 @@
+package duke;
+
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Helper class for user interface, handles
+ * user input and outputting to the user.
+ */
+public class Ui {
+    private final String name;
+
+    private final Scanner scanner;
+
+    /**
+     * Initialise duke.Ui handler.
+     * @param name Name of chatbot.
+     */
+    public Ui(String name) {
+        this.name = name;
+        this.scanner = new Scanner(System.in);
+    }
+
+    /**
+     * Initialise UI. Prints banner.
+     */
+    public void init() {
+        System.out.println("____________________________________________________________");
```

Comment:
Small thing but perhaps it would be better to turn the underline into a constant so it can be easily reused elsewhere

Reply:
Thank you! That makes sense, will do so.